### DISCUSSION TITLE
Add additional informtions for error message which misses uses/run

### DISCUSSION THREAD

**Discussion by Fettpet** [community/community#9140] [Actions]
**Describe the enhancement**
When I created a custom github acton, I ran into the error message "Error : .github#L1 every step must define a `uses` or `run` key". The error message misses the step which hasn't a 'uses' or 'run' key.  The debugging process would easier if the message contains the line number and/or the name of the step which suffers of a uses/run key.

**Code Snippet**
For example these code snippet is wrong. It is hard to see, that the problem is the additional '-' by uses.
```
      - name: Checkout source
      - uses: actions/checkout@v2
```

**Additional Information**

The enhancement was first created in [runner ](https://github.com/actions/runner/issues/1480)

**Comment by ethomson** [comment]
Thanks for the feedback, I agree that we could improve this.
